packages/core/upload/admin/src/components/EditAssetDialog/PreviewBox/index.js

```diff
@@ -73,11 +73,11 @@ export const PreviewBox = ({
     const nextAsset = { ...asset, width, height };
     const file = await produceFile(nextAsset.name, nextAsset.mime, nextAsset.updatedAt);
 
-    await editAsset(nextAsset, file);
+    const updatedAsset = await editAsset(nextAsset, file);
 
     // Making sure that when persisting the new asset, the URL changes with width and height
     // So that the browser makes a request and handle the image caching correctly at the good size
-    const optimizedCachingImage = createAssetUrl(asset);
+    const optimizedCachingImage = createAssetUrl(updatedAsset);
```

Comment:
ℹ️ createAssetUrl uses the "updated_at" attribute to handle image caching. The problem is that I was using the previous asset and so, the URL was always the same => cache was hit in this scenario